### PR TITLE
Add cpln config

### LINKED ISSUE #12: CPLN Demo
@ahangarha can we have this live as a demo of CPLN?

### PR DESCRIPTION
**Pull request by ahangarha**
Deployment is live at https://rails-8t7fhx09hf0j0.cpln.app

Closes #12

### CODE REVIEW

File: .controlplane/controlplane.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  react-rails-example-app:
+    <<: *common
+  tutorial-app:
+    <<: *common
+  # QA Apps are like Heroku review apps, but the use `prefix` so you can run a commmand like
+  # this to create a QA app for the tutorial app.
+  # `cpl setup gvc postgres redis rails -a qa-tutorial-app-pr-1234`
+  qa-tutorial-app:
+    <<: *common
+    prefix: true
```
Comment (ahangarha) [inline .controlplane/controlplane.yml]: I kept these lines similar to RWRT project. But I don't think we need them. Should I remove them?

Update: I just removed them

File: .controlplane/entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+# Runs before the main command
+
+# No action is required for this project to be done before running the app.
```
Comment (ahangarha) [inline .controlplane/entrypoint.sh]: This entrypoint file doesn't have any use here. I kept it to keep the structure.

Update: the file is removed

**Comment by ahangarha** [comment]
Is this finalized? @Judahmeek

**Review by justin808 — APPROVED** [review]
all good, but CSR is???

yes, I know


but we should spell out CSR and SSR please!

**Comment by ahangarha** [comment]
@justin808 Let's fix this issue in a separate PR: #14.

**Pull request merged**